src/traces/ohlc/calc.js

```diff
@@ -69,8 +70,14 @@ function calcCommon(gd, trace, x, ya, ptFunc) {
         var ci = c[i];
 
         if(xi !== undefined && oi !== undefined && hi !== undefined && li !== undefined && ci !== undefined) {
-            // increasing carries over from previous if ci===oi
-            increasing = (ci === oi) ? increasing : ci > oi;
+            if(ci === oi) {
+                // if open == close, look for a change from the previous close
+                if(cPrev !== null && ci !== cPrev) increasing = ci > cPrev;
+                // else (c === cPrev or cPrev is null) no change
+            }
+            else increasing = ci > oi;
```

Comment:
What does this fix exactly?

Reply:
This is code that used to be in [`_isIncreasing`](https://github.com/plotly/plotly.js/pull/2561/commits/84c36a91091aa5524e6b2a9f0788766b622146de#diff-e872430731145abebb1f8f55bf310949L108) that I didn't copy over right the first time, but it affects fallbacks when `open===close`, as tested [here](https://github.com/plotly/plotly.js/pull/2561/files#diff-7deb9e75965ca15196b29a1f7a4a9210R586)